eval.go

```diff
@@ -372,6 +372,10 @@ func splitKeys(s string) (keys []string) {
 }
 
 func update(app *app) {
+	// Remove the current menu buffer
```

Comment:
I added this here as, for example, in Vim when you delete a character while calling for completition it removes the completiton menu.